mitiq/zne/scaling/layer_scaling.py

```diff
@@ -24,6 +24,15 @@ def layer_folding(
 ) -> cirq.Circuit:
     """Applies a variable amount of folding to select layers of a circuit.
 
+    Note that this method only works for the univariate extrapolation methods. 
+    It allows a user to choose which layers in the input circuit will be scaled
+    The layerwise folding required for multivariate extrapolation is different
+    as the layers in the input circuit have to be scaled in a specific pattern.
+    The required specific pattern for multivariate extrapolation does not allow
+    a user to provide a choice of which layers to fold. If you would prefer to
+    use a multivariate extrapolation method for unitary
+    folding, use the functions available in `mitiq/lre` instead. 
```

Comment:
When this PR is close to being merged, replace `mitiq/lre` with the `seealso` directive from Sphinx. 

https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-seealso